### ISSUE TITLE
the setting of random seed

### ISSUE THREAD

**Issue by Arya87** [nicklashansen/tdmpc#8]
The result of my experiment on Reacher-easy is only 318±172(random seed is 1 to 5), which the article is 628±105. All settings are the same as in the article.  Maybe it is because of the setting of the random seed. Could you share with us the exact value of the random seed? Thank you very much!

**Comment by nicklashansen** [comment]
Thank you for your interest! The experiments in Table 1 and Figure 4 of our paper differ slightly in experimental setup. Figure 4 experiments use an action repeat of 2 in all tasks, whereas we used an action repeat of 4 for Reacher Easy in Table 1 following previous work. Additionally, the two settings also use slightly different hyperparameters. Table 1 uses the same learning rate and batch size as in our state-based experiments (lr=1e-3, batch_size=512), whereas Figure 4 uses lr=3e-4 and batch_size=256 to be consistent with the methods that we compare to. Apologies if these details are not made clear in the paper. I will close this issue for now, but feel free to reopen if it is not resolved.
